src/transformers/integrations.py

```diff
@@ -492,10 +492,23 @@ def setup(self, args, state, model):
             if hasattr(model, "config") and model.config is not None:
                 model_config = model.config.to_dict()
                 combined_dict = {**model_config, **combined_dict}
+            # remove params that are too long for MLflow
+            for name, value in list(combined_dict.items()):
+                # internally, all values are converted to str in MLflow
+                if len(str(value)) > MLFLOW_MAX_PARAM_VAL_LENGTH:
+                    logger.warning(
+                        "Trainer is attempting to log a value of "
+                        '"%s" for key "%s" as a parameter. '
```

Comment:
Can we use a f-string here? It's clearer to read :-)

Reply:
I used this `%s` approach because:
1. It is used in other parts of code in this module (see `TensorBoardCallback`), so it just looks uniform.
2. It is generally recommended, though for reasons not applicable here (https://docs.python.org/3/howto/logging-cookbook.html#formatting-styles)

Would you like me to change logging message formatting throughout the integration.py module as well?

Reply:
The link you mention does mention f-strings and says it's a perfectly valid solution ;-) With Python 3.5 officially dead, I would very much like to use them more consistently as they are way easier to read. Feel free to change other instances in the integration file.